var/spack/repos/builtin/packages/prime95/package.py

```diff
@@ -6,7 +6,7 @@
 from spack.package import *
 
 
-class Prime95(AutotoolsPackage):
+class Prime95(Package):
```

Comment:
Curious.  Why switch from `AutotoolsPackage`?

Reply:
AutotoolsPackage forces an "autoreconf" stage that isn't valid here.
I could have overwritten that function, but felt it was simple to just switch to "Package" class instead.

Thanks